core/src/main/kotlin/com/cognifide/apmt/junit/WireMockExtension.kt

```diff
@@ -0,0 +1,15 @@
+package com.cognifide.apmt.junit
+
+import com.github.tomakehurst.wiremock.WireMockServer
+import org.junit.jupiter.api.extension.ExtensionContext
+import org.junit.jupiter.api.extension.ParameterContext
+import org.junit.jupiter.api.extension.ParameterResolver
+
+class WireMockExtension : ParameterResolver {
+    override fun resolveParameter(parameterContext: ParameterContext?, extensionContext: ExtensionContext?): Any {
```

Comment:
Fix formmating here.

Reply:
It's formatted according to the IntelliJ's Kotlin style guide :)